codec_specs.md

```diff
@@ -88,6 +88,10 @@ The values of `BlockAddID` that are 2 of greater have no semantic meaning, but s
 associate the `BlockMore Element` with a `BlockAdditionMapping` of the associated Track.
 See (#block-additional-mapping) on Block Additional Mappings for more information.
 
+The value of 4 for `BlockAddID` SHOULD NOT be used when `BlockAddIDType` is not 4,
+as some legacy demuxers consider a block with `BlockAddID` of 4 as ITU T.35 metadata
+without checking `BlockAddIDType`.
```

Comment:
What legacy demuxers? I'm only now adding support for this in ffmpeg and i'm making sure to check BlockAddIDType too, for example.
The problem could arise with WebM parsers, where 4 is used for HDR10+ in ITU-T T35 as BlockAddID, but BlockAdditionMapping elements are not supported.

Reply:
Yes, I was thinking to e.g. WebM parsers.

> but BlockAdditionMapping elements are not supported.

And some parsers may just ignore them when checking ITU-T T35 as it may be common code with Matroska parser. Else it would not be useful to force 4 for Matroska specs, and we could remove this mandatory BlockAddID for ITU-T T35 in Matroska.

Also as 4 is mandatory for ITU-T T35 also in Matroska, a remuxer would have to remap a former stream with BlockAddID of 4 if it needs to inject a ITU-T T35 stream, better to recommend to avoid.

Note that I indicated "SHOULD NOT", not "MUST NOT", if someone wants to use 4, well, fine...

Reply:
>Yes, I was thinking to e.g. WebM parsers.

Might be worth mentioning that alongside "legacy demuxers", then. But either way it looks good.